### PR TITLE
Try to reset console to switch to correct tty

### PR DESCRIPTION
**Pull request by tinawang123**
Try to reset console to switch to correct tty

- Related ticket: https://progress.opensuse.org/issues/105121
- Verification run: 
 http://openqa.suse.de/tests/8004435#
http://openqa.suse.de/tests/8004431
http://openqa.suse.de/tests/8004433
http://openqa.suse.de/tests/8004434

### CODE REVIEW

File: tests/migration/version_switch_upgrade_target.pm [hunk 1]
```diff
@@ -47,6 +47,7 @@ sub run {
 
     record_info('Version', 'VERSION=' . get_var('VERSION'));
     reconnect_mgmt_console if is_pvm;
+    reset_consoles_tty;
```
Comment (coolgw) [inline tests/migration/version_switch_upgrade_target.pm:50]: if is_pvm then
reconnect_mgmt_console 
else
reset_consoles_tty

Comment (tinawang123) [reply tests/migration/version_switch_upgrade_target.pm:50]: I checked powerVM, Don't need this condition, it still can work.

**Comment by lemon-suse** [comment]
Please write the PR's background, introduce the history in the description, it will help people to understand why need this PR.

**Review by coolgw — APPROVED** [review]

**Review by chcao — APPROVED** [review]
LGTM

**Pull request merged**